docs/source/en/tasks/video_classification.mdx

```diff
@@ -127,7 +127,12 @@ Next up, you will derive the set of labels present in the dataset. Also, create
 * `id2label`: maps the integers to class names. 
 
 ```py 
->>> class_labels = sorted({str(path).split("/")[2] for path in all_video_file_paths})
+>>> from pathlib import Path
+
+>>> base_path = Path(file_path)
```

Comment:
I think this might have to be `UCF101_subset`. The dataset when downloading with `hf_hub_download` I believe will download to the cache folder -- `file_path` -- whereas the tar files will be extracted to the local directory.